CITATION.cff

```diff
@@ -0,0 +1,66 @@
+﻿# Content: Citation File Format (CFF) file for webKnossos
+cff-version: 1.2.0
+title: "WEBKNOSSOS"
+message: >-
+  If you use this software, please cite it using the
+  metadata from this file.
+type: software
+authors:
+  - family-names: Boergens
+    given-names: Kevin M
```

Comment:
Shouldn't all authors be listed here as well?

Reply:
Doesn't have an impact on the Github citation feature since only the "preferred-citation" key is used for that, but yes, it is cleaner